apps/devhub/forms.py

```diff
@@ -515,6 +515,10 @@ class NewAddonForm(AddonUploadForm):
         help_text=_(u'Add-ons that are side-loaded will be code reviewed by '
                     u'Mozilla before they are signed and are held to a higher '
                     u'quality standard.'))
+    is_manual_review = forms.BooleanField(
+        initial=False,
+        required=False,
+        label=_lazy(u'Submit my add-on for manual review.'))
```

Comment:
So, do we need this after all? We won't be using it in the python view, so maybe we don't?